### PR TITLE
Rename file integrity fields

### PR DESCRIPTION
**Pull request by andrewkroh**
This creates three groups of fields in the file integrity module `event`, `hash`, and `file`.

It also changes `dataset.module` to `event.module` for all modules.

See data.json files for sample events.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Split the audit.kernel and audit.file metricsets into their own modules
   named auditd and file_integrity, respectively. This change requires
   existing users to update their config. {issue}5422[5422]
+- Renamed file_integrity module fields.
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: can you add teh PR number?

File: auditbeat/docs/fields.asciidoc [hunk 2]
```diff
@@ -2144,91 +2144,89 @@ Image labels.
 These are the fields generated by the file_integrity module.
 
 
+[float]
+=== `event.action`
```
Comment (ruflin) [inline auditbeat/docs/fields.asciidoc]: Interesting field, we should discuss this more.

File: auditbeat/module/file_integrity/metricset.go [hunk 3]
```diff
@@ -21,7 +21,7 @@ const (
 	bucketName    = "file.v1"
 
 	// Use old namespace for data until we do some field renaming for GA.
-	namespace = "audit.file"
+	namespace = "."
```
Comment (ruflin) [inline auditbeat/module/file_integrity/metricset.go]: I assume that means it ignores the namespace?

File: metricbeat/mb/event.go [hunk 4]
```diff
@@ -47,11 +47,16 @@ func (e *Event) BeatEvent(module, metricSet string, modifiers ...EventModifier)
 	}
 
 	if len(e.MetricSetFields) > 0 {
-		prefix := e.Namespace
-		if prefix == "" {
-			prefix = module + "." + metricSet
+		switch e.Namespace {
+		case ".":
+			// Add fields to root.
+			b.Fields.DeepUpdate(e.MetricSetFields)
```
Comment (ruflin) [inline metricbeat/mb/event.go]: Here we go :-)

**Comment by ruflin** [comment]
@andrewkroh I think this PR needs a make fmt

**Comment by andrewkroh** [comment]
Update the CHANGELOG and fmt'ed Metricbeat.

File: auditbeat/module/file_integrity/_meta/fields.yml [hunk 5]
```diff
@@ -2,157 +2,161 @@
...
-          type: keyword
-          description: >
-            The path to the file. This is an non-analyzed field that is useful
-            for aggregations.
-
-      - name: target_path
+    - name: path
+      type: text
+      description: The path to the file.
+      multi_fields:
```
Comment (ruflin) [inline auditbeat/module/file_integrity/_meta/fields.yml]: Nice

**Pull request merged**